services/acceptance/services/errortelemetry.test.js

```diff
@@ -119,7 +124,10 @@ describe('Error Telemetry service acceptance tests', () => {
           json: true,
           body: emptyLog
         })
-          .then(res => assert.ok(res))
+          .then(res => {
+            assert.ok(res);
+            assert.equal(res.status, 'OK', 'response.status should be OK but got: ' + res.status);
```

Comment:
One of the things @batmat taught me was about [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) in JavaScript (as of ECMAScript 6) which you may find helpful.

Reply:
Is templating preferred over string concatenation?

Reply:
I would recommend it, it definitely makes life easier IMHO.

There's no way to enforce that convention that I have found with ESLint however.